src/Program.cs

```diff
@@ -8,99 +9,115 @@ namespace ms_continuus
     internal static class Program
     {
         private static readonly Config Config = new();
+        private static readonly Api Api = new();
+        private static readonly BlobStorage BlobStorage = new();
 
         private static async Task DeleteWeeklyBlobs()
         {
             var olderThan = Utility.DateMinusDays(Config.WeeklyRetention);
             Console.WriteLine($"Deleting blobs with retention='weekly' older than {olderThan}");
-
-            var blobStorage = new BlobStorage();
-            await blobStorage.EnsureContainer();
-            await blobStorage.DeleteArchivesBefore(olderThan, "weekly");
+            await BlobStorage.DeleteArchivesBefore(olderThan, "weekly");
         }
 
         private static async Task DeleteMonthlyBlobs()
         {
             var olderThan = Utility.DateMinusDays(Config.MonthlyRetention);
             Console.WriteLine($"Deleting blobs with retention='monthly' older than {olderThan}");
+            await BlobStorage.DeleteArchivesBefore(olderThan, "monthly");
+        }
 
-            var blobStorage = new BlobStorage();
-            await blobStorage.EnsureContainer();
-            await blobStorage.DeleteArchivesBefore(olderThan, "monthly");
+        private static async Task<bool> DownloadAndUpload(Migration migration, int index)
+        {
+            Migration migStatus = await Api.MigrationStatus(migration.Id);
+            var exportTimer = 0;
+            const int sleepIntervalSeconds = 30;
+            while (migStatus.State != "exported")
```

Comment:
Do we have an enum, instead of comparing with a string?

Reply:
fixed